tools/lxdclient/client.go

```diff
@@ -153,13 +144,12 @@ func Connect(cfg Config, verifyBridgeConfig bool) (*Client, error) {
 		}
 	}
 
-	var bridgeName string
+	newClient := lxd.NewClient(raw)
 	if remoteID == remoteIDForLocal && verifyBridgeConfig {
 		// If this is the LXD provider on the localhost, let's do an extra check to
 		// make sure the default profile has a correctly configured bridge, and
 		// which one is it.
-		bridgeName, err = verifyDefaultProfileBridgeConfig(raw, networkAPISupported, defaultProfile)
-		if err != nil {
+		if err := newClient.VerifyDefaultBridge(defaultProfile); err != nil {
```

Comment:
I feel like a lot of this code is just all historic because LXD didn't track networks.
Now that it does, we should probably prefer the new LXD apis for network listing and engage with it in that fashion rather than thinking directly about bridges.